### PR TITLE
Add EFCore.AppInsights package

### PR DESCRIPTION
**Pull request by smitpatel**

### CODE REVIEW

**Comment by smitpatel** [comment]
@anpete @ajcvickers - Can you verify if category = ship & optimizedCache = include are correct values?

**Comment by smitpatel** [comment]
@ryanbrandenburg

File: packages/packages.csv [hunk 1]
```diff
@@ -119,6 +119,7 @@ Microsoft.Data.Sqlite,ship,include
 Microsoft.Data.Sqlite.Core,ship,include
 Microsoft.DotNet.Watcher.Tools,ship,exclude
 Microsoft.EntityFrameworkCore,ship,include
+Microsoft.EntityFrameworkCore.ApplicationInsights,ship,include
```
Comment (pranavkm) [inline packages/packages.csv:122]: cc @JunTaoLuo

Comment (JunTaoLuo) [reply packages/packages.csv:122]: Is this added here: https://github.com/aspnet/MetaPackages/blob/dev/build/common.props#L75?

Comment (smitpatel) [reply packages/packages.csv:122]: No

**Comment by JunTaoLuo** [comment]
Is this a duplicate of https://github.com/aspnet/Coherence/pull/211?

**Comment by smitpatel** [comment]
Closed this since #211 is already merged.

**Comment by JunTaoLuo** [comment]
Still needs the update in Metapackages

**Comment by smitpatel** [comment]
@bricelam

**Comment by bricelam** [comment]
@glennc Said these may not end up in the Metapackage and to just file an issue.

**Comment by bricelam** [comment]
Filed aspnet/MetaPackages#116

**Pull request closed without merging**